src/modules/mount/main.py

```diff
@@ -307,6 +307,12 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
                                         mount_option) != 0:
                 libcalamares.utils.warning("Cannot mount {}".format(device))
 
+def enable_swap_partition(swap_device):
+    try:
+        subprocess.check_call(["swapon", swap_device])
```

Comment:
Don't use subprocess, instead use the built-in Calamares functions as described in the modules readme.

Reply:
For example?

Reply:
`host_env_process_output(["swapon", swap_device])`

It throws `subprocess.CalledProcessError` if there is an error.